contributing/workflow.qmd

```diff
@@ -649,14 +654,104 @@ Cookbook that have that link, and then you can do a search-replace to
 update them all at once.
 
 ## Cookbook Editing
-TODO Add these 3 sections
+
+*TODO Add these 3 sections*
 
 ### Fix a typo
 
 ### Edit a chapter
 
 ### Add a chapter
 
+### Review a pull request
+
+Any pull request (PR) must be reviewed before it can be merged into the
+Cookbook. If you know someone who is familiar with the content you're
+proposing to add, request their review.
+
+The following notes are adapted from a coworking session in which Amy
+Steiker and Julie Lowndes reviewed a Cookbook [pull
+request](https://github.com/NASA-Openscapes/earthdata-cloud-cookbook/pull/171)
+from Andy Barrett. ([video](https://youtu.be/2_F5Kr6gM1I), 19 min). Some
+generic mechanics of reviewing a pull request in GitHub are demonstrated
+in a screenshare ([video](https://youtu.be/ZiG4-d5vT30), 7 min 41 sec).
+
+In reviewing a pull request with lots of changes, it can be helpful to
+have windows open to view both the GitHub browser and the Hub showing
+the Cookbook preview.
+
+From the pull request page in GitHub browser, look at the elements of
+the pull request.
+
+![](images/Screenshot%202023-07-10%20at%2012.55.55%20PM.png)
+
+-   Start with Conversation tab:
+    -   We can see all commits and comments on what Andy has worked on
```

Comment:
```suggestion
    -   We can see all commits and comments on what Andy has worked on
    -   This is where we can add PR reviewers by clicking the gear icon next to "Reviewers" at the top right corner of this tab.
```

Reply:
I'm not sure how to get the formatting right here (ensuring the second line stays indented.